port_status_yaml.py

```diff
@@ -18,10 +18,17 @@ class Comment:
     class Source:
         repo: str
         commit: str
+    @dataclass
+    class Label:
+        name: str
+        color: str
+        text_color: str = '000000'
```

Comment:
I don't think this belongs here; we don't need it in the `yaml`, and this file should act as a specification for what's actually in the yaml file.

Reply:
It's just I don't know how to call a python function in the `.j2` file, so I need to assign `label.text_color = ...` somewhere.

Reply:
You can add the function to `template_env.filters` (see elsewhere in the file), and then apply it as `label.color | text_color`.

Reply:
nvm, done.